deploy/docker/gaiahub.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+TASK=$1
+WHICH=$(which docker)
```

Comment:
`which` isn't available on all *nix systems these days.  Recommend replacing with `command -v`, which is POSIX-ly correct.

Reply:
fixed.